Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM ubuntu:14.04
+FROM progrium/busybox
```

Comment:
I have mixed feelings about it too... But how many backdoors can you hide in a 5MB base image? :)

I'm not exactly sure how the hash thing works... but I don't think this image has a hash yet...

Anyways, this is an issue when we rebuild the image. I'm sure something more credible will be around next time... Besides looking at this guys github profile, I'm not so worried :)